.github/workflows/ci.yml

```diff
@@ -99,8 +99,36 @@ jobs:
         run: |
           python -m pip install --upgrade pip pipx
           python -m pip install -r requirements/requirements_build.txt
-      - name: Build, install and run
+      - name: Build wheel and sdist
         run: |
           python -m build && python -m twine check dist/*
-          pipx install dist/ansys_templates-0.1.dev0-py3-none-any.whl
-          pipx run --spec dist/ansys_templates-0.1.dev0-py3-none-any.whl ansys-templates --help
+
+  release:
+    name: Release
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+    needs: [style, docs, tests, build]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
```

Comment:
This should be newer than 3.7 so we can avoid another release at the end of its [lifespan](https://peps.python.org/pep-0537/#lifespan).